### ISSUE TITLE
Publish under @react-three/scissor

### ISSUE THREAD

**Issue by gsimone** [pmndrs/react-three-scissor#3]

**Comment by drcmda** [comment]
i think this will be simple enough to express as a small drei component since many of the underlying requirements are now being fulfilled by fiber. @FarazzShaikh and i have been churning through some prototypes and looking very very good. the last state of that component was probably the best thing in the fiber/three eco system i've seen. 😀

**Comment by FarazzShaikh** [comment]
Yeah let’s go ahead and archive/delete this repo and put that in the readme to avoid confusion when people come here
